public/assets/sass/pages/get-involved/council.scss

```diff
@@ -28,3 +28,67 @@ section {
     padding: 8px;
   }
 }
+
+.mailing-list-form {
+  max-width: var(--content-width);
+  margin: 40px auto;
+  padding: 5px;
+  background-color: #f0f0f0;
+  border: 1px solid #e0e0e0;
+  border-radius: 8px;
+
+  section {
+    text-align: center;
+
+    h3 {
+      font-size: 24px;
+      margin-bottom: 20px;
+      color: #333;
+    }
+
+    form {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      padding: 20px 0;
+
+      label {
+        font-size: 16px;
+        margin-bottom: 10px;
+        font-weight: bold;
+      }
+
+      input[type="email"] {
+        width: 100%;
+        max-width: 400px;
+        padding: 12px;
+        font-size: 16px;
+        border: 1px solid #ccc;
+        border-radius: 4px;
+        margin-bottom: 15px;
+      }
+
+      .error-message {
+        color: red;
+        font-size: 14px;
+        margin-top: 0;
+        display: none;
+      }
+
+      button {
+        padding: 10px 20px;
+        font-size: 16px;
+        background-color: #da69ad;
```

Comment:
same here, could we use `var(--mathsoc-pink)`? there's probably a bunch of other spots i haven't caught

Reply:
Since you mentioned the `pink-button` class, I removed my CSS for `button` and just used `pink-button` for the button.